pkg/keyspace/tso_keyspace_group.go

```diff
@@ -146,6 +155,30 @@ func (m *GroupManager) Close() {
 	m.wg.Wait()
 }
 
+func (m *GroupManager) allocNodesForDefaultKeyspaceGroup() {
+	defer logutil.LogPanic()
+	defer m.wg.Done()
+	ticker := time.NewTicker(allocNodesForDefaultKeyspaceGroupInterval)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-m.ctx.Done():
+			return
+		case <-ticker.C:
+		}
+		kg, err := m.GetKeyspaceGroupByID(utils.DefaultKeyspaceGroupID)
+		replica := m.GetNodesNum()
+		if err == nil && kg != nil && len(kg.Members) >= replica {
```

Comment:
add all nodes for the default keyspace group, because we do not support offline nodes in keyspace balancer, but we test delete tso node primary in test. If only one replica, some test will fail.